setup.py

```diff
@@ -13,6 +13,7 @@
         "libpebble2",
         "openaps",
         "tzlocal",
+        "python-dateutil"
```

Comment:
Nit: please add a trailing comma like the others.

https://stackoverflow.com/questions/11597901/why-does-python-allow-a-trailing-comma-in-list

Reply:
fixed